docs/01.GETTING-STARTED.md

```diff
@@ -81,6 +81,36 @@ For example the cross-compile to RaspberryPi 2 is something like this:
 python tools/build.py --toolchain=cmake/toolchain_linux_armv7l.cmake
 ```
 
+**Use system memory allocator**
+
+```bash
+python tools/build.py --system-allocator=on --jerry-libc=off
+```
+
+*Note*: System allocator is only supported on 32 bit systems.
+
+**Enable 32bit compressed pointers**
+
+```bash
+python tools/build.py --cpointer-32bit=on
+```
+
+*Note*: There is no compression/decompression on 32 bit systems, if enabled.
+
+**Change default heap size (512K)**
+
+```bash
+python tools/build.py --mem-heap=256K
+```
+
+If you would like to use more than 512K, then you must enable the 32 bit compressed pointers.
+
+```bash
+python tools/build.py --cpointer-32bit=on --mem-heap=1024K
+```
+
+*Note*: Memory allocator of JerryScript will allocate the full heap, when the engine starts.
```

Comment:
This isn't completely correct, as our memory allocator has no responsibility to allocate the heap. The heap will be allocated statically at compile time.